docs/src/gs/advanced.md

```diff
@@ -0,0 +1,16 @@
+# Advanced Options
+
+## Disable OCI Client initialization on startup
+
+CAPOCI supports setting OCI principals at [cluster level][cluster-identity], hence CAPOCI can be
+installed without providing OCI user credentials. The following environment variable need to be exported
+to install CAPOCI without providing any OCI credentials.
+
+   ```shell
+   export INIT_OCI_CLIENTS_ON_STARTUP=false
+   ```
```

Comment:
Should we call out failure mode if this is set, but the cluster level principals isn't set? I know we default to it enabled, but it might be nice to have a "look for X in the logs" if not set at the cluster level.

Reply:
Fixed, please check

Reply:
awesome. thank you 😊